vnext/Microsoft.ReactNative/ABIViewManager.cpp

```diff
@@ -53,9 +53,13 @@ const wchar_t *ABIViewManager::GetName() const {
   return m_name.c_str();
 }
 
-xaml::DependencyObject ABIViewManager::CreateViewCore(int64_t) {
-  auto view = m_viewManager.CreateView();
-  return view;
+xaml::DependencyObject ABIViewManager::CreateViewCore(int64_t, winrt::Microsoft::ReactNative::JSValueObject &props) {
+  if (auto viewCreateProps = m_viewManager.try_as<IViewManagerCreateWithProperties>()) {
+    auto view = viewCreateProps.CreateViewWithProperties(
+        MakeJSValueTreeReader(winrt::Microsoft::ReactNative::JSValue(std::move(props.Copy()))));
```

Comment:
>props.Copy() [](start = 79, length = 12)

Could you add a bug for me to avoid copy in situations like this? I think we must be able to create JSValueTreeReader directly for the JSValueObject.

Reply:
#7150